sdk/devcenter/developer-devcenter-rest/package.json

```diff
@@ -0,0 +1,127 @@
+{
+  "name": "@azure-rest/developer-devcenter",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0",
```

Comment:
why are we release GA version as the api version is still preview ?

Reply:
Updated to 1.0.0-beta.1, thanks for catching this!